### PR TITLE
Application Directory & File Structure

### PR DESCRIPTION
**Pull request by arianacofone**
Morning Gents!

The file structure is all completed now. Let me know what feedback you have!

### CODE REVIEW

File: app/config/sql/user/all.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+//This file and folder will hold all sql commands to be performed on the database
```
Comment (nferner) [inline app/config/sql/user/all.sql]: Hey, sql files can be commented out using /\* for multiple lines or for a single line you can use --

File: migrations/migrations.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+// This file will set up how the database migrations will occur
```
Comment (nferner) [inline migrations/migrations.sql]: /\* for multi line comment or -- for single line

File: app/config/sql/user/all.sql (hunk 1 above)
Comment (arianacofone) [reply app/config/sql/user/all.sql]: Thank you! Correcting now.

File: migrations/migrations.sql (hunk 2 above)
Comment (arianacofone) [reply migrations/migrations.sql]: Super! Correcting and pushing the changes.

**Review by nferner — APPROVED** [review]
Thanks Ariana, for sql you can use -- for single line comments and the good 'ol js /\* for multi line comments

File: app/config/sql/user/all.sql (hunk 1 above)
Comment (tbongiorno1) [reply app/config/sql/user/all.sql]: great stuff ariana

Comment (arianacofone) [reply app/config/sql/user/all.sql]: Thanks for reviewing! Merging now :)

**Pull request merged**